plugins/plugin-upload/frontend/public/javascripts/countly.views.js

```diff
@@ -1,230 +1,106 @@
-                    id: "plugin-upload-widget-drawer",
-                    template: src,
-                    title: jQuery.i18n.map["plugin-upload.upload-title"],
-                });
-
-
-                myDropzone = new Dropzone("#plugin-upload-drop", {
-                    url: '/',
-                    autoQueue: false,
-                    param_name: "new_plugin_input",
-                    parallelUploads: 0,
-                    maxFiles: 1,
-                    addedfile: function(file) {
-                        if (check_ext(file.name)) {
-                            myDropzone.disable();
-                            $('#plugin-upload-drop').removeClass('file-hovered');
-                            $('#plugin-upload-drop').addClass('file-selected');
-                            $(".dz-filechosen").html('<div class="dz-file-preview"><p><i class="fa fa-archive" aria-hidden="true"></i></p><p class="sline">' + countlyCommon.encodeHtml(file.name) + '</p><p class="remove" id="remove-files"><i class="fa fa-trash"  aria-hidden="true"></i> ' + jQuery.i18n.map["plugin-upload.remove"] + '</p></div>');
-                            $('#upload-new-plugin').removeClass('mydisabled');
-                        }
-                    },
-                    dragover: function() {
-                        $('#plugin-upload-drop').addClass('file-hovered');
-                    },
-                    dragleave: function() {
-                        $('#plugin-upload-drop').removeClass('file-hovered');
-                    }
-                });
-
-                $(window).on('resize', function() {
-                    resizePluginUploadFileBox();
-                });
-
-                //pull out plugin-upload form
-                $("#show-plugin-upload").on("click", function() {
-                    resizePluginUploadFileBox();
-                    self.plugin_upload_drawer.open();
-                });
-                /** function resizes upload box. */
-                function resizePluginUploadFileBox() {
-                    var newPluginUploadFileBoxHeight = $("#plugin-upload-widget-drawer").height() - 50;
-                    $("#plugin-upload-drop").height((newPluginUploadFileBoxHeight < 180) ? 180 : newPluginUploadFileBoxHeight);
+/*global CountlyHelpers, countlyGlobal, jQuery, CV, countlyVue */
+
+(function() {
+    var PluginUploadDrawer = countlyVue.views.create({
+        template: CV.T('/plugin-upload/templates/drawer.html'),
+        data: function() {
+            return {
+                uploadData: {
+                    _csrf: countlyGlobal.csrf_token
+                },
+            };
+        },
+        props: {
+            controls: { type: Object }
+        },
+        methods: {
+            handleUploadSuccess: function(result) {
+                var aa = result.split('.');
+                if (aa.length === 2 && aa[0] === 'Success') {
+                    this.$emit('updloaded', aa[1]);
```

Comment:
A typo here "updloaded". It'd work though, because the handler below is also `@updloaded` :slightly_smiling_face: